packages/manager/modules/nasha/src/components/edit-name/edit-name.template.html

```diff
@@ -0,0 +1,27 @@
+<div class="oui-modal oui-modal_shadow">
+    <form name="editNameForm" data-ng-submit="$ctrl.submit()" novalidate>
+        <oui-modal
+            data-primary-label="{{:: ('nasha_dashboard_edit_name_primary' | translate) }}"
+            data-primary-disabled="!$ctrl.canSubmit"
+            data-secondary-label="{{:: ('nasha_dashboard_edit_name_secondary' | translate) }}"
+            data-secondary-action=":: $ctrl.dismiss()"
+            data-on-dismiss=":: $ctrl.dismiss()"
+            data-loading="$ctrl.isSubmitting"
+            data-heading="{{:: ('nasha_dashboard_edit_name_title' | translate) }}"
+        >
+            <oui-field
+                data-label="{{:: ('nasha_dashboard_edit_name_label' | translate) }}"
+            >
+                <input
+                    class="oui-input"
+                    type="text"
+                    name="name"
+                    id="name"
+                    data-ng-model="$ctrl.name"
+                    data-ng-pattern="/^[^<>]+$/"
```

Comment:
Maybe move this into a constant.

Reply:
@ghyenne  This pattern is meant to block html chars, in order to avoid HTML injection. It is used in a lot of places in the repository. Thus the work to factorize it, can be quickly heavy as it implies to change all the concerned ng-pattern.
One more point to consider is that it will implies injecting the module storing it, also in a lot of places.

If we choose to do it, it should be done in a separate task

Reply:
Fixed here https://github.com/ovh/manager/pull/6868/files#diff-727ac56c7351e941f33ef2d990eab67d27a6fd502c189413cff5cdd0572a32e2